code/modules/library/lib_items.dm

```diff
@@ -316,3 +316,53 @@
 	else
 		user << "<font color=red>No associated computer found. Only local scans will function properly.</font>"
 	user << "\n"
+
+/obj/item/weapon/chisel
+	name = "soapstone"

```

Comment:
Give this a proper name instead of forced reference that makes no sense in context.